packages/upload-api/src/types/delegations.ts

```diff
@@ -5,6 +5,16 @@ interface ByAudience {
 }
 export type Query = ByAudience
 
+export interface RevocationMeta {
```

Comment:
I'm not sure what does `Meta` imply here, maybe `RevocationDetails` would be less confusing name